### ISSUE TITLE
🚀Make TCP port configurable

### ISSUE THREAD

**Issue by canterberry** [blockmason/lndr#163]
Currently, the LNDR app server is hard-coded to listen on TCP port 80. Unless something like `authbind` is used, this requires super-user privileges on the host. This service should instead listen on an unprivileged port by default, and be configurable to listen on any port. I propose port `7402` for the default, since it most closely resembles the letters `LNDR`.

**Comment by canterberry** [comment]
It's still a work in progress -- I keep getting tripped up on Haskell syntax. You could probably knock this out in a couple minutes, though.

**Comment by aupiff** [comment]
Yeah, I'd happily make these changes. Would make my local test script much less annoying by letting me run it w/o sudo.
